NTR - Update 'using-utils.md' (#706)
shopware/docs: guides/plugins/plugins/administration/using-utils.md

# Using utility functions

> Utility functions in the Shopware 6 Administration are registered to [the Shopware object](the-shopware-object.md) and are therefore accessible everywhere in the Administration. They provide many useful shortcuts for common tasks, see all of them [here](../../../../resources/references/core-reference/administration-reference/utils.md) in our reference section.

Reviewer: ```suggestion
Utility functions in the Shopware 6 Administration are registered to [the Shopware object](the-shopware-object.md) and are therefore accessible everywhere in the Administration. They provide many useful [shortcuts](../../../../resources/references/core-reference/administration-reference/utils.md) for common tasks.
```

## Prerequisites

> All you need for this guide is a running Shopware 6 instance, the files and preferably a registered module. Of course you'll have to understand JavaScript. However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.

Reviewer: ```suggestion
All you need for this guide is a running Shopware 6 instance, the files, a registered module, and a good understanding of JavaScript.
```

## Accessing the utility functions

> In this very short example, we'll show you how to use a one of the utility functions, in this case the `capitalizeString` function. The `capitalizeString` function calls the [`lodash capitalize`](https://lodash.com/docs/4.17.15#capitalize) function, which - as the name implies - capitalizes strings.

Reviewer: ```suggestion
Let us see how to use one of the utility functions — for example, `capitalizeString` function. As the name implies, the `capitalizeString` function capitalizes strings by calling the [`lodash capitalize`](https://lodash.com/docs/4.17.15#capitalize) function.
```
